Sources/Plasma/PubUtilLib/plInterp/plAnimTimeConvert.cpp

```diff
@@ -1086,14 +1074,14 @@ plAnimTimeConvert& plAnimTimeConvert::PlayToPercentage(float percent)
 
 void plAnimTimeConvert::RemoveCallback(plEventCallbackMsg* pMsg)
 {
-    int idx = fCallbackMsgs.Find(pMsg);
-    if( idx != fCallbackMsgs.kMissingIndex )
+    auto idx = std::find(fCallbackMsgs.cbegin(), fCallbackMsgs.cend(), pMsg);
+    if (idx != fCallbackMsgs.cend())
     {
-        hsRefCnt_SafeUnRef(fCallbackMsgs[idx]);
-        fCallbackMsgs.Remove(idx);
+        hsRefCnt_SafeUnRef(*idx);
+        fCallbackMsgs.erase(idx);
     }
```

Comment:
I have a tendency to reuse variable names on massive refactors like this (mostly to avoid accidentally shadowing other variables), but you're right, these should be fixed even though I did plenty more like this on previous reviews ;)